packages/react/src/Form/Checkbox/Client/Checkbox.tsx

```diff
@@ -0,0 +1,69 @@
+import { CheckboxItem, CheckboxItemOption } from "./CheckboxItem";
+
+export enum CheckboxModes {
+  classic = "classic",
+  default = "default",
+  inline = "inline",
+}
+
+type Props = {
+  name: string;
+  values: string[];
+  options: CheckboxItemOption[];
+  mode?: keyof typeof CheckboxModes;
+  disabled?: boolean;
+  onChange: React.ChangeEventHandler;
+};
+
+const baseClassNameMode = "af-form-client__checkbox";
+
+const getContainerClassNameMode = (mode: Props["mode"]) => {
+  switch (mode) {
+    case CheckboxModes.classic:
+      return `${baseClassNameMode}-container`;
+    case CheckboxModes.inline:
+      return `${baseClassNameMode}-container-inline`;
+    default:
+      return `${baseClassNameMode}-container-custom`;
+  }
+};
+
+const getItemClassNameMode = (mode: Props["mode"]) => {
+  switch (mode) {
+    case CheckboxModes.classic:
+      return baseClassNameMode;
+    case CheckboxModes.inline:
+      return `${baseClassNameMode}-inline`;
+    default:
+      return `${baseClassNameMode}-custom`;
+  }
+};
+
+export const Checkbox = ({
+  name,
+  options,
+  values,
+  disabled,
+  mode = CheckboxModes.classic,
+  onChange,
+  ...otherProps
+}: Props) => (
+  <div className={getContainerClassNameMode(mode)}>
+    {options.map((option) => (
+      <CheckboxItem
+        {...otherProps}
+        {...option}
+        onChange={onChange}
+        key={option.value}
+        className={getItemClassNameMode(mode)}
+        icon={option.icon}
+        label={option.label}
+        description={option.description}
+        checked={values ? values.indexOf(option.value) >= 0 : false}
+        name={name}
+        disabled={option.disabled || disabled}
+        value={option.value}
```

Comment:
Option passe déjà ces paramètres au dessus normalement
```suggestion
        className={getItemClassNameMode(mode)}
        checked={values ? values.indexOf(option.value) >= 0 : false}
        name={name}
        disabled={option.disabled || disabled}
```

Reply:
C'est exact 😅